dojo/tools/whitesource/parser.py

```diff
@@ -67,7 +67,7 @@ def _build_common_output(node, lib_name=None):
                             topfix_node.get('fixResolution')
                         )
                 except Exception as e:
-                    print("Error handling topFix node. {}").format(e)
+                    ("Error handling topFix node. {}").format(e)
```

Comment:
Is a logger missing here?

Reply:
wow, nice spot. am struggling lately with visual studio code is it's so damn slow with python code that even copy and pasting is laggy

Reply:
I reverted it back to what it was.... which is invalid python code...